### PR TITLE
[ticket/13955] Fix WebFontConig to load web fonts correctly

### PR DESCRIPTION
**Pull request by kinerity**
Fix overall_header's WebFontConfig section to load fonts correctly
in all browsers.

PHPBB3-13955

### CODE REVIEW

**Comment by vinny** [comment]
https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/styles/prosilver/template/overall_header.html#L31~L44

I visited the Google Fonts and it seems that the code has been updated to:
```javascript
<script type="text/javascript">
  WebFontConfig = {
    google: { families: [ 'Open+Sans:600:cyrillic-ext,latin,greek-ext,greek,vietnamese,latin-ext,cyrillic' ] }
  };
  (function() {
    var wf = document.createElement('script');
    wf.src = ('https:' == document.location.protocol ? 'https' : 'http') +
      '://ajax.googleapis.com/ajax/libs/webfont/1/webfont.js';
    wf.type = 'text/javascript';
    wf.async = 'true';
    var s = document.getElementsByTagName('script')[0];
    s.parentNode.insertBefore(wf, s);
  })(); </script>
```


Furthermore, the change must also be made to the file <a href="https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/styles/prosilver/template/simple_header.html#L10~L23">simple_header.html</a>

**Comment by kinerity** [comment]
Ah, so it has. I'll push an update.

**Comment by kinerity** [comment]
What gives - why does it keep failing?

**Comment by vinny** [comment]
is in progress, not failing.

**Comment by kinerity** [comment]
Check 14508.5

**Comment by vinny** [comment]
I see now.

**Comment by marc1706** [comment]
That's a random failure. Restarted that job now and tests pass.

File: phpBB/styles/prosilver/template/overall_header.html [hunk 1]
```diff
@@ -28,19 +28,19 @@
 -->
 
 <!-- IF S_ALLOW_CDN -->
-<script>
+<script type="text/javascript">
```
Comment (marc1706) [inline phpBB/styles/prosilver/template/overall_header.html]: The type is not needed. Please undo this change.

File: phpBB/styles/prosilver/template/simple_header.html [hunk 2]
```diff
@@ -7,19 +7,19 @@
 <title>{SITENAME} &bull; <!-- IF S_IN_MCP -->{L_MCP} &bull; <!-- ELSEIF S_IN_UCP -->{L_UCP} &bull; <!-- ENDIF -->{PAGE_TITLE}</title>
 
 <!-- IF S_ALLOW_CDN -->
-<script>
+<script type="text/javascript">
```
Comment (marc1706) [inline phpBB/styles/prosilver/template/simple_header.html]: Same as above.

File: phpBB/styles/prosilver/template/overall_header.html [hunk 3]
```diff
@@ -28,19 +28,19 @@
...
-			families: ['Open Sans:n6']
-		}
+		google: { families: [ 'Open+Sans:600:cyrillic-ext,latin,greek-ext,greek,vietnamese,latin-ext,cyrillic' ] }
 	};
-
-	(function(d) {
-		var wf = d.createElement('script'), s = d.scripts[0];
-		wf.src = 'https://ajax.googleapis.com/ajax/libs/webfont/1.5.18/webfont.js';
-		wf.async = true;
+	(function() {
```
Comment (marc1706) [inline phpBB/styles/prosilver/template/overall_header.html]: Please undo the changes to this part. We decided to have them this way for a reason. Please only change the families setting above.

File: phpBB/styles/prosilver/template/overall_header.html [hunk 4]
```diff
@@ -28,19 +28,19 @@
 -->
 
 <!-- IF S_ALLOW_CDN -->
-<script>
+<script type="text/javascript">
 	WebFontConfig = {
-		google: {
-			families: ['Open Sans:n6']
-		}
+		google: { families: [ 'Open+Sans:600:cyrillic-ext,latin,greek-ext,greek,vietnamese,latin-ext,cyrillic' ] }
```
Comment (marc1706) [inline phpBB/styles/prosilver/template/overall_header.html]: The families part should be on a new line like previously.

File: phpBB/styles/prosilver/template/simple_header.html [hunk 5]
```diff
@@ -7,19 +7,19 @@
...
 	WebFontConfig = {
-		google: {
-			families: ['Open Sans:n6']
-		}
+		google: { families: [ 'Open+Sans:600:cyrillic-ext,latin,greek-ext,greek,vietnamese,latin-ext,cyrillic' ] }
```
Comment (marc1706) [inline phpBB/styles/prosilver/template/simple_header.html]: See above comments for this part of the header.

**Comment by kinerity** [comment]
Changes made. Awaiting tests.

**Comment by kinerity** [comment]
Same failure as last time. Can you restart the test?
